### PR TITLE
Add tabs movement and fix tab command with correct counting

### PR DESCRIPTION
**Pull request by rebornix**
Yay! We love PRs! 🎊

Please include a description of your change & check your PR against this list, thanks:

- [x] Commit message has a short title & issue references
- [x] Commits are squashed 
- [x] It builds and tests pass (e.g `gulp tslint`)

1. Correct the behavior of `{count}`.
2. Add tabs movement support with newly added Code API.

### CODE REVIEW

File: src/cmd_line/subparsers/tab.ts [hunk 1]
```diff
@@ -3,22 +3,22 @@
 import * as node from "../commands/tab";
 import {Scanner} from '../scanner';
 
-function parseCount(args: string): number {
+function parseCount(args: string): number | undefined {
```
Comment (johnfn) [inline src/cmd_line/subparsers/tab.ts]: 👍

File: src/cmd_line/commands/tab.ts [hunk 2]
```diff
@@ -45,25 +46,56 @@ export class TabCommand extends node.CommandBase {
...
+          vscode.commands.executeCommand("workbench.action.closeActiveEditor");
+          break;
+        }
+
+        if (this._arguments.count === 0) {
+          // Wrong paramter
+          break;
+        }
+
+        // TODO: Close Page {count}. Page count is one-based.
```
Comment (johnfn) [inline src/cmd_line/commands/tab.ts]: Are you leaving this intentionally incomplete?

**Comment by johnfn** [comment]
1 comment, then LGTM!

File: src/cmd_line/commands/tab.ts (hunk 2 above)
Comment (rebornix) [inline src/cmd_line/commands/tab.ts]: yup, it's saying we require additional Code support. Will fill in this part whens it' possible.

Comment (johnfn) [inline src/cmd_line/commands/tab.ts]: Ahh, gotcha.

**Comment by infogulch** [comment]
It looks like ts compile step failed. See [src/cmd_line/commands/tab.ts](https://github.com/VSCodeVim/Vim/commit/caf3f0e8597af04f1eb51f59a5c2eed980b131f8#diff-507eb0edaa9bf1d2e186296409f07e73R94). Also [travis](https://travis-ci.org/VSCodeVim/Vim/jobs/147111465#L687).

**Pull request merged**